tools/project.py

```diff
@@ -225,15 +223,15 @@ def make_flags_str(flags: Optional[Union[str, List[str]]]) -> str:
 # Load decomp-toolkit generated config.json
 def load_build_config(
     config: ProjectConfig, build_config_path: Path
-) -> Optional[Dict[str, Any]]:
+) -> dict[str, Any | None]:
```

Comment:
```suggestion
) -> dict[str, Any] | None:
```
Unless I’m misreading